Build/jenkins/Jenkinsfile_x86-64_linux

```diff
@@ -51,16 +51,16 @@ pipeline {
             steps {
 				timestamps {
                 	echo 'Deploying....'
-                	step([$class: "TapPublisher", testResults: "**/*.tap"])
```

Comment:
The deploy step is now empty.  In the future, I imagine us uploading test results to a cloud-based DB.  If you think that action would live here, then we can keep this step, otherwise, perhaps we remove it.